diff --git a/sites/all/modules/ubercart/uc_roles/uc_roles.module b/sites/all/modules/ubercart/uc_roles/uc_roles.module
index 56cf9c8..a081576 100644
--- a/sites/all/modules/ubercart/uc_roles/uc_roles.module
+++ b/sites/all/modules/ubercart/uc_roles/uc_roles.module
@@ -310,7 +310,7 @@ function uc_roles_user_presave(&$edit, $account, $category) {
     $edit['roles'][$edit['new_role_add']] = _uc_roles_get_name($edit['new_role_add']);
 
     // Reset the new role form.
-    $edit['new_role'] = $edit['new_role_add'] = $edit['new_role_add_qty'] = $edit['new_role_add_granularity'] = NULL;
+    // $edit['new_role'] = $edit['new_role_add'] = $edit['new_role_add_qty'] = $edit['new_role_add_granularity'] = NULL;
   }
 
   // Check if any temporary role actions were taken.
